spec/latest/core/core.jsonld

```diff
@@ -431,6 +432,16 @@
       "@id": "hydra:template",
       "@type": "rdf:Property",
       "label": "template",
+      "comment": "A templated string with placeholders. The literal's datatype indicates the template syntax; if not specified, hydra:rfc6570Template is assumed.",
+      "seeAlso": "hydra:rfc6570Template",
+      "domain": "hydra:IriTemplate",
+      "range": "rdfs:Literal",

```

Comment:
Assume it's a literal property, and thus perform literal operations. Although it's probably mostly handy for humans writing program code against this property.

While I do think properties should at least be either literals or objects, I'm not fighting for this one. Whatever resolution you prefer is fine with me.